docs/sw-update-conf.json

```diff
@@ -14,110 +14,81 @@
     },
     "restart": "environment"
   },
-  "octoprint": {
-    "pip": "https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
-    "prerelease": false,
-    "develop":{
-      "prerelease": true
-    },
-    "beamos_date": {
-      "2021-06-11": {
+  "modules": {
+    "octoprint": {
+      "type": "github_release",
+      "develop": {
+        "type": "github_commit"
+      },
+      "beta": {
+        "prerelease_channel": "mrbeam2-{tier}",
+        "prerelease": true
+      },
+      "alpha": {
+        "prerelease_channel": "mrbeam2-{tier}",
         "prerelease": true
       }
-    }
-  },
-  "mrbeam": {
-    "name": " MrBeam Plugin",
-    "repo": "MrBeamPlugin",
-    "restart": "octoprint",
-    "pip": "https://github.com/mrbeam/MrBeamPlugin/archive/{target_version}.zip",
-    "beamos_date": {
-      "2021-06-11": {
-        "branch": "mrbeam2-{tier}-buster",
-        "develop": {
-          "branch": "develop"
+    },
```

Comment:
Can we assume here that stable will pull from the release directly but dev, alpha, and beta according to the above configuration?

Reply:
if there is a additional configuration for a tier this will override the dedault config of the module